### DISCUSSION TITLE
Add More Ways To Sort Rides

### DISCUSSION THREAD

**Discussion by WhenPenguinsAttack** [OpenRCT2/OpenRCT2#15691] [Feature requests] [answered]
Add more ways to sort rides, such as by nausea and intensity.  This would really be helpful in multiplayer, so we can build First Aid rooms by other players' coasters if

**Comment by duncanspumpkin** [comment]
Such as from 5878:

Price
Size
Excitement
Intensity
Nausea
Reversed Alphabetical order

**Comment by J-PIE-314** [comment]
This feature has been implemented at some point in development.

**Reply by tupaschoal** [reply, marked as answer]
Seems like solved by #12793
